src/VCR/RequestMatcher.php

```diff
@@ -1,102 +0,0 @@
-<?php
-
-namespace VCR;
-
-/**
- * Collection of matcher methods to match two requests.
- */
-class RequestMatcher
```

Comment:
is there a risk of BC breaks when removing this class? or is it only ever used internally?